frontend/wallet/src/render/views/settings/SettingsPage.re

```diff
@@ -122,14 +129,25 @@ module Version = {
   };
 };
 
+type ownedAccounts =
+  Account.t = {
+    locked: option(bool),
+    publicKey: PublicKey.t,
+    balance: {. "total": int64},
+  };
+  
 module AccountSettingsItem = {
   [@react.component]
-  let make = (~publicKey) => {
+  let make = (~publicKey, ~account) => {
```

Comment:
Does account contain the pubkey?
It also looks pretty similar to the ownedAccounts type, any reason we're not using that?

Reply:
it does- should we just use account instead of passing in both publicKey and account?

Reply:
changed it!